### ISSUE TITLE
Extension methods don't appear to exist when using a jar

### ISSUE THREAD

**Issue by markyys** [manifold-systems/manifold#171]
Hello, another newbie question again. This appears to be very similar (possibly identical) to the issue raised in #25, except that I only have one project, not two.

tl;dr: is it required that any extension methods are defined in a standalone project, then included as a dependency of another project if I want to use them from a compiled jar?

The scenario I have is that I have some project which defines some extension class (i.e. TestProject.extensions.java.lang.String.StringExt). I was able to compile and run everything manually with no issue, but eventually the manual dependency management became cumbersome, so I decided to try to switch to Maven.

Using Maven I can compile everything with no issue, but when I try to execute the jar I produce, it fails with a "method [...] is undefined for the type String" error.

I followed all the steps in the docs, including those mentioned in #25, and my pom.xml appears to look exactly how it is supposed to (annotationProcessorPaths, manifestEntries Contains-Sources, etc.), but I still get this error.

I have attached my pom.xml for reference.

[pom.xml.txt](https://github.com/manifold-systems/manifold/files/4457039/pom.xml.txt)

**Comment by markyys** [comment]
Okay, once again I am a victim of inexperience and insufficient Java/Maven documentation.

tl;dr: I'm using JDK 13, but the `maven.compiler.release` property in my pom.xml was set to 11. This caused many things to fail in non-obvious ways.

Changing that to 13 caused everything to work as advertised.
